### PR TITLE
[ELY-2828] Replace wildcard imports with specific imports inside X500PrincipalBuilderTest.java

### PR DESCRIPTION
**Pull request by Samith10**
https://issues.redhat.com/browse/ELY-2828

### CODE REVIEW

**Comment by fjuma** [comment]
@Samith10 Just a small comment, looks like your changes for ELY-2826 also ended up in this PR.

**Comment by rsearls** [comment]
1+ looks good

**Review by luck3y — APPROVED** [review]

**Comment by luck3y** [comment]
@Samith10 FYI you seem to have changes from a previous commit included in this PR as well, do you need some help removing that one? (If you look at the changes there are two files changed).

**Comment by luck3y** [comment]
You can use something like:

$ git rebase -i HEAD~2
use the instructions on the screen to drop the unwanted commit (change the letter to 'd') then save.

[Note corrected git command from 'git commit rebase' to 'git rebase'.

File: x500/base/src/test/java/org/wildfly/security/x500/X500PrincipalBuilderTest.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 package org.wildfly.security.x500;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEqua;
```
Comment (PrarthonaPaul) [inline x500/base/src/test/java/org/wildfly/security/x500/X500PrincipalBuilderTest.java]: I think this is supposed to be `assertEquals`?

**Review by PrarthonaPaul — CHANGES_REQUESTED** [review]
Hello @Samith10 Thank you! 
I have left a small comment that can help resolve the error you see in the CI

File: x500/base/src/test/java/org/wildfly/security/x500/X500PrincipalBuilderTest.java (hunk 1 above)
Comment (Samith10) [reply x500/base/src/test/java/org/wildfly/security/x500/X500PrincipalBuilderTest.java]: Yeah, sorry just making tiny changes to figure out merge issues

**Comment by PrarthonaPaul** [comment]
Hello @Samith10 
It seems like you have 4 commits, 2 related to ELY-2828 and 2 related to 2826. 
Could you please merge the ones related to ELY-2828 and drop the ones associated with ELY2826? Please feel free to let us know if you have any questions.

**Pull request closed without merging**